bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/IntesisConfiguration.java

```diff
@@ -23,4 +23,5 @@
 public class IntesisConfiguration {
```

Comment:
Please split this in two classes if the Things have different configurations.

Reply:
Done